ChessHeatmap/modules/YearResult/YearResultView.swift

```diff
@@ -10,10 +10,15 @@ import SwiftUI
 struct YearResultView: View {
     @Environment(\.chessClient) var chessClient
     @State var username: String = ""
-    @State var year: Int = 2023
+    @State var year: Int = Calendar.current.component(.year, from: Date())
     @State var results: OrderedGames?
     @State var searching = false
     @State var message: String? = nil
+    
+    var availableYears: [Int] {
```

Comment:
nit: make this a `@State private var availableYears` for now as a computed property will get run every single time `body` is computed as well. Creating dates are kind of expensive.

Reply:
Got the error "property wrapper cannot be applied to a computed property" from the compiler when adding `@State` and `private` to this existing computed property. 

So, instead I switched it to a function that only gets called when the `VStack` in the `body` appears 👍🏻 Lmk if you had something else in mind.